src/components/common/useFormikResetOnEvent.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+
+// reusable component to hook into form locations.
+// We have a discard dialog in the library-actions-center that needs to know about every other form in the app, and trigger the reset onContinue
+// We could have that onContinue logic emit an event, and this component should add the listener on each form that needs to blank.
+const useFormikResetOnEvent = (formik) => {
```

Comment:
Would be nice to make it globally reusable, instead of having a separate one in madie-cql-library and madie-measure. Maybe in the future.

Reply:
Well we already have this for madie-measure in its component so I just created a separate component for this in cql library and I don't think we will use it any further in any other place so it should be okay to leave it here